opcua/server/history.py

```diff
@@ -89,17 +97,17 @@ def save_node_value(self, node, datavalue):
         if count and len(data) > count:
             data = data[-count:]
 
-    def read_node_history(self, node, start, end, nb_values):
+    def read_node_history(self, node_id, start, end, nb_values):
         cont = None
-        if node not in self._datachanges:
+        if node_id not in self._datachanges:
             print("Error attempt to read history for a node which is not historized")
             return [], cont
         else:
             if end is None:
                 end = datetime.now() + timedelta(days=1)
             if start is None:
                 start = ua.DateTimeMinValue
-            results = [dv for dv in self._datachanges[node] if start <= dv.ServerTimestamp <= end]
+            results = [dv for dv in self._datachanges[node_id] if start <= dv.ServerTimestamp <= end]
             if nb_values:
                 if start > ua.DateTimeMinValue and len(results) > nb_values:
                     cont = results[nb_values + 1].ServerTimestamp
```

Comment:
This line needs to be fixed. It should be:
```python
cont = results[nb_values + 1].ServerTimestamp
```
Otherwise 1 array member will be skipped due to the list starting at 0.